buddyboss/buddyboss-platform
# PROD-773 Fatar error when Clearing default data from BuddyBoss Tools

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2749  .

### How to test the changes in this Pull Request:

=> Activate Forum discussions component first
=> Import data related to Forum discussions like Discussions/Replies
=> Deactivate Forum discussions component
=> The again go to BuddyBoss > Tools > Click on Clear default data
=> The Fatal Error should not appear anymore in this fix as stated in the issue

### Proof Screenshots or Video

https://www.loom.com/share/609d53c82adf4fa391844b16006815cb


### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
